package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
@jreidinger sorry it is not about merging changes from SP2 into CASP but from CASP back to SP2 ... I mean all the changes from `3.1.210.1` to `3.1.210.6`